packages/tools/package.json

```diff
@@ -48,34 +50,34 @@
     "@cardstack/mock-auth": "0.11.10",
     "@cardstack/models": "0.11.10",
     "@cardstack/test-support": "0.11.10",
-    "broccoli-asset-rev": "^2.4.5",
-    "ember-cli": "^3.0.0",
-    "ember-cli-dependency-checker": "^2.1.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.2",
-    "ember-cli-inject-live-reload": "^1.4.1",
-    "ember-cli-qunit": "^4.1.0",
-    "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^2.0.0",
+    "@ember/optional-features": "^0.6.3",
+    "broccoli-asset-rev": "^2.7.0",
+    "ember-cli": "~3.4.3",
+    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
+    "ember-cli-inject-live-reload": "^1.8.2",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^2.1.0",
     "ember-data": "^3.4.2",
-    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
     "ember-inflector": "^2.1.0",
-    "ember-let": "^0.7.1",
-    "ember-load-initializers": "^1.0.0",
+    "ember-load-initializers": "^1.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
-    "ember-source-channel-url": "^1.0.1",
-    "ember-try": "^0.2.23",
-    "eslint": "^5.8.0",
-    "eslint-plugin-ember": "^5.0.0",
-    "eslint-plugin-node": "^5.2.1",
-    "loader.js": "^4.2.3",
+    "ember-resolver": "^5.0.1",
+    "ember-source": "~3.4.0",
+    "ember-source-channel-url": "^1.1.0",
+    "ember-try": "^1.0.0",
+    "eslint": "^5.9.0",
+    "eslint-plugin-ember": "^5.2.0",
+    "eslint-plugin-node": "^7.0.1",
+    "loader.js": "^4.7.0",
     "qunit-dom": "^0.8.0",
     "squishable-container": "^0.1.0"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "6.* || 8.* || >= 10.*"
```

Comment:
since we only run CI on Node.js 8 we should consider adjusting this to `8.* || >= 10.*`

Reply:
Sorry, I overlooked this one, I'll create another issue.